### PR TITLE
PLANET-5705 - Update form inputs UI

### PR DESCRIPTION
**Pull request by mleray**
### Description
- [PLANET-5705](https://jira.greenpeace.org/browse/PLANET-5705): update form input styles (border, fonts, colors)
- [PLANET-5512](https://jira.greenpeace.org/browse/PLANET-5512): add label/placeholder animation

### Related PRs
- master-theme: https://github.com/greenpeace/planet4-master-theme/pull/1253
- gutenberg-blocks: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/454

### CODE REVIEW

File: src/layout/_forms.scss [hunk 1]
```diff
@@ -19,140 +19,202 @@
   input[type="checkbox"] {
     position: absolute;
     opacity: 0;
+    pointer-events: none;
```
Comment (comzeradd) [inline src/layout/_forms.scss:22]: Just curious, why is this needed for checkboxes?

**Review by comzeradd — APPROVED** [review]
Looks good :+1:

File: src/layout/_forms.scss (hunk 1 above)
Comment (mleray) [reply src/layout/_forms.scss:22]: Otherwise since the input is full width, and even if the description isn't that long, clicking anywhere on the line will check the input which looks strange 😬

Comment (mleray) [reply src/layout/_forms.scss:22]: It's the same for radio buttons 🙂
